src/dokumentasjonsbehov/DokumentasjonsbehovApp.tsx

```diff
@@ -96,22 +101,32 @@ const DokumentasjonsbehovApp = () => {
                 <Sidetittel>Dokumentasjon til søknaden</Sidetittel>
                 <div className="seksjon">
                   <Normaltekst>
-                    Det ser ut til at det mangler noe dokumentasjon i søknaden
-                    du har sendt oss. Hvis du i mellomtiden har sendt oss dette,
-                    kan du se bort fra denne meldingen.
+                    {hentIngressTekst(manglendeVedlegg.length > 0)}
                   </Normaltekst>
+                  {manglendeVedlegg.length === 0 && (
```

Comment:
Burde man ha `manglendeVedlegg.length > 0 || vedlegg.length > 0 ` ? Og att ettersendelseknappen fjernes fra ManglendeVedlegg-komponenten?

Reply:
Nei, i følge skissene skal ettersendingsknappen ligge under manglende vedlegg-seksjonen i de tilfellene der det mangler vedlegg.

Reply:
Nevnte for Tuva igår att den ikke vises hvis man ikke mangler vedlegg, men fortsatt har sendt inn vedlegg. Då kan det være så att man fortsatt ønsker å sende inn vedlegg.

Noe vi burde prata om idag på møtet tenker jeg

Reply:
Ja, det gjør den jo nå? Tuva har en ny skisse på figma for dette som er implementert i denne PR-en: https://www.figma.com/file/aIqFpogtD2CHqu6UZbqoUK/S%C3%B8knadsdialoger-EF

Reply:
Ja stemmer det, tenkte vi ikke skulle vise den hvis man kun viser "du har allerde sendt inn"